### DISCUSSION TITLE
Examples of using urql with multiple graphql schemas

### DISCUSSION THREAD

**Discussion by Yuripetusko** [urql-graphql/urql#2674] [Q&A] [answered]
We are displaying multiple very different products within the app that have their own gql endpoints with their own distinct schemas. I am looking for some examples on how to handle this with urql / react

**Comment by JoviDeCroock** [comment, marked as answer]
https://github.com/FormidableLabs/urql/discussions/1523#discussioncomment-575307

Generally you can arrange this globally with i.e a custom exchange or on a hook level through the operationContext
